plugin/yt_dlp_plugins/extractor/getpot_bgutil.py

```diff
@@ -62,4 +77,8 @@ def _get_pot_via_script(self, script_path, visitor_data, data_sync_id):
         # the JSON response is always the last line
         script_data_resp = result.stdout.splitlines()[-1]
         self._logger.debug(f"_get_pot_via_script response = {script_data_resp}")
-        return json.loads(script_data_resp)['poToken']
+        try:
+            return json.loads(script_data_resp)['poToken']
+        except:
+            raise RequestError("Error parsing JSON response from _get_pot_via_script")
```

Comment:
is this necessary? `json.loads` should deal with all kinds of strings generated from `JSON.stringify`

Reply:
Incase the output from the script is unexpected not a JSON.stringified value. Right now we expect that the JSON is the last line of stdout, if we accidentally print something to stdout after the JSON, we might have issues.